app/Http/Controllers/Auth/ConfirmPasswordController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Providers\RouteServiceProvider;
+use Illuminate\Foundation\Auth\ConfirmsPasswords;
+
+class ConfirmPasswordController extends Controller
+{
+    /*
+    |--------------------------------------------------------------------------
+    | Confirm Password Controller
+    |--------------------------------------------------------------------------
+    |
+    | This controller is responsible for handling password confirmations and
+    | uses a simple trait to include the behavior. You're free to explore
+    | this trait and override any functions that require customization.
+    |
+    */
+
+    use ConfirmsPasswords;
+
+    /**
```

Comment:
very nice BRO

Reply:
![image](https://github.com/SamZuckerberg/marketplace/assets/31242039/3a381cd1-d2c3-471a-8078-a9a00aabc5e7)

Reply:
this is Juliano?

Reply:
no its you 🗡️